src/contractIntegration/interaction/web3Provider/Web3ProviderClam.ts

```diff
@@ -0,0 +1,57 @@
+import Web3 from "web3";
+import { IdentityManager } from "../../../indentityManager";
+import IInteractionConfig from "../IInteractionConfig";
+import IWeb3Provider from "./IWeb3Provider";
+import { injectable, inject } from "tsyringe";
+
+@injectable()
+class Web3ProviderClam implements IWeb3Provider {
```

Comment:
@HectorGerardoZV Please rename the class to Web3Provider same with the file name

Reply:
Done